Add setting to show camera views

vexofp opened the pull request:
Closes #17

ironmonk88 commented:
So v9 changed the way styles are being used.
Unfortunately I didn't get to this before I converted to v9.
Would it be possible to update and resolve the conflicts?

vexofp commented:
Updated and tested on v9.

I think I also found and fixed a minor bug where the `#loading` bar was not being shown on the common display as intended.
